agent/README.md

```diff
@@ -0,0 +1,24 @@
+# Agent based installer environment
+
+This folder contains all the files and assets required to deploy a cluster
+using the agent-based installer approach.
+
+# Quickstart
+
+Use the following configuration in your `config_<user>.sh` to deploy a 
+compact cluster:
+
+    # Configure ipv4 environment
+    export IP_STACK=v4
+    export NETWORK_TYPE=OpenShiftSDN
+
+    # Configure master nodes specs
+    export MASTER_DISK=120
```

Comment:
Is this a typo?  the default is 30G per master, 120G is going to fill up even large virthost disks pretty fast?

Reply:
This is a requirement that is enforced by Assisted Installer otherwise it wont start the install. However, the defaults can be overidden using the environment variable `HW_VALIDATOR_REQUIREMENTS` - docs for this are [here](https://github.com/openshift/assisted-service/blob/4b4f0f45d897430037039a9282225bf6bd02acfe/docs/dev/hardware-requirements.md)

Reply:
Can we get libvirt to create them as sparse QCOWs?